projects/openshift.py

```diff
@@ -35,20 +37,37 @@
 @task(name='install-deps')
 def install_deps():
     install_packages([
-        'docker-io',
+        'docker',
+        'docker-registry',
         'git',
         'golang',
         'screen',
         'vim',
     ])
+    # required on CentOS7
+    sudo('yum -y update device-mapper')
 
 
 @task
-def deploy(proxy=''):
+def deploy():
     install_deps()
 
-    enable_services(['docker.socket'])
+    enable_services(['docker.service'])
+
     enable_tcp_ports([8443, 8444])
 
-    sudo('usermod -a -G docker {0}'.format(env.user))
+    sudo('groupadd docker')
```

Comment:
I thought so, but it's not added automatically and the script fails without it:
'usermod: group 'docker' does not exist'